atmos_model.F90

```diff
@@ -1456,7 +1491,7 @@ subroutine update_atmos_chemistry(state, rc)
           ib = i + Atm_block%isc - 1
           nb = Atm_block%blkno(ib,jb)
           ix = Atm_block%ixp(ib,jb)
-          hpbl(i,j)   = IPD_Data(nb)%IntDiag%hpbl(ix)
+          hpbl(i,j)   = IPD_Data(nb)%Tbd%hpbl(ix)
```

Comment:
May I ask why hpbl is changed from diag_type to Tbd type?

Reply:
This was required, because it is not a diagnostic quantity but a prognostic variable for MYNN. In other words, to prevent a reset every `fhour` and make it persistent.